server/interview/interview.service.js

```diff
@@ -0,0 +1,39 @@
+const db = require('../helpers/db');
+
+const questionService = require('../question/question.service');
+const userService = require('../users/user.service');
+
+const Interview = db.Interview;
+
+module.exports = {
+    createInterview,
+    updateInterview
+}
+
+async function createInterview(userId, difficulty) {
+    let interview = new Interview();
+    let participant = await userService.getById(userId);
+    let question = await questionService.findARandomQuestionByDifficulty(difficulty);
+    interview.participants.push(participant);
+    interview.questions.push(question);
+    await interview.save();
+
+    return interview;
+}
+
+async function updateInterview(userId, difficulty, interviewId) {
```

Comment:
Is this to only add a new user to the interview object? If, so I'd change it's name, since this sounds like you can do more than just that

Reply:
Yeah it is to add new user to an existing interview. I think I would change it to "addParticipantToAnInterview".